lib/storage/app_constants.dart

```diff
@@ -7,6 +7,9 @@ const LatLng sgw = LatLng(45.495944, -73.578075);
 const LatLng loyola = LatLng(45.4582, -73.6405);
 const LatLng hBuilding = LatLng(45.497094, -73.578797);
 const LatLng jmsbBuilding = LatLng(45.495549, -73.579036);
+const LatLng fgBuilding = LatLng(45.494344, -73.578442);
```

Comment:
Can we somehow use building data for this to avoid duplicating data?

Reply:
Will be done during refactoring